rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -83,14 +85,16 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RMServiceImpl.class);
 
-    private ClientExecutor executor;
     private DefaultHttpClient httpClient;
 
+    private ExecutorService threadPool;
+
     @Override
     public void init() {
         loadProperties();
+
         httpClient = HttpUtils.createDefaultExecutor();
-        executor = new ApacheHttpClient4Executor(httpClient);
+        threadPool = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 8);
```

Comment:
Why 8? Could this property be defined as a private static final String with a name that explain what it is?